Lib/test/test_opcache.py

```diff
@@ -481,6 +484,513 @@ def f(x, y):
             f()
 
 
+class TestRacesDoNotCrash(unittest.TestCase):
+    # Careful with these. Bigger numbers have a higher chance of catching bugs,
+    # but you can also burn through a *ton* of type/dict/function versions:
+    ITEMS = 1 << 10
```

Comment:
Any reason why these need to be powers of 2? If not, maybe use `1000` instead.

Reply:
It was useful when I was trying to dial in a good value while being aware of exactly how many versions I was using. I can change them, though.